bibedit/record_cdfnote.xml

```diff
@@ -0,0 +1,69 @@
+<!-- BibEdit-Template-Name: cdfnote -->
+<!-- BibEdit-Template-Description: CDF internal note -->
+<record>
+  <datafield tag="035" ind1=" " ind2=" ">
+    <subfield code="a">CDFNote-####</subfield>
+    <subfield code="9">CDF</subfield>
+  </datafield>
+  <datafield tag="037" ind1=" " ind2=" ">
+    <subfield code="a">CDF-####</subfield>
```

Comment:
These are mandatory fields - leaving them with #s is exactly as ugly as not entering them.